.prettierignore

```diff
@@ -1,3 +1,4 @@
 *.js
 package.json
-package-lock.json
\ No newline at end of file
+package-lock.json
+CHANGELOG.md
```

Comment:
why is changelog.md ignored in prettier? Did you have problems with this? In my projects I do not have changelog.md excluded from prettier and have no problems.

Reply:
In my projects I live with the `-` and as the git ``pre-commit` hook activated by husky runs `prettier` the changelog is always formatted the right way. And AFAIK for markdown viewer `-` or `*` doesn't make any difference.